11ty/definitions/peanut-gallery.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Peanut Gallery
+slug: peanut-gallery
+flag:
+  level: avoid
+  text: Racist Term
+defined: true
+speech: noun
+reading:
+  - text: 'Is Peanut Gallery a racist term?'
+    href: http://www.todayifoundout.com/index.php/2015/03/origin-phrase-peanut-gallery/
+  - text: 'Dictionary.com'
+    href: https://www.dictionary.com/browse/peanut-gallery
+alt_words:
+  - disrupters
+  - hecklers
+---
+
+Used to refer to those giving unsolicited (and unvalued) advice.
+
+## Issues
+
+Before microphones and opera glasses were common, balcony seats were the worst in the theater. In Vaudevill theaters, the term "Peanut Gallery" became a nickname for these seats where Black people were forced to sit. The least expensive snack served at the theatre would often be peanuts, which the patrons would be rumored to throw at the performers on stage to convey their disapproval. Phrases such as "no comments from the peanut gallery" or "quiet in the peanut gallery" are extensions of the name.
```

Comment:
Tighten and move this section up.

Add something like, "The term perpetuates and reinforces the historic silencing of Black voices and speaking of them in a pejorative nature."